win32ss/user/ntuser/sysparams.c

```diff
@@ -1369,10 +1376,10 @@ SpiGetSet(UINT uiAction, UINT uiParam, PVOID pvParam, FLONG fl)
             return SpiGetInt(pvParam, &gspv.bFontSmoothing, fl);
 
         case SPI_SETFONTSMOOTHING:
-            gspv.bFontSmoothing = uiParam ? TRUE : FALSE;
+            gspv.bFontSmoothing = (uiParam == 2);
             if (fl & SPIF_UPDATEINIFILE)
             {
-                SpiStoreSzInt(KEY_DESKTOP, VAL_FNTSMOOTH, uiParam ? 2 : 0);
+                SpiStoreSz(KEY_DESKTOP, VAL_FONTSMOOTHING, (uiParam == 2) ? L"2" : L"0");
```

Comment:
Isn't the SpiStoreSzInt function fine ?

Reply:
fine, but it's a minor optimization

Reply:
"Minor optimization" is something to be avoided. Readable code has priority.